kubeproxy/config/manager/manager.yaml

```diff
@@ -22,13 +22,20 @@ spec:
       labels:
         control-plane: controller-manager
     spec:
+      hostNetwork: true
       containers:
       - command:
         - /manager
         args:
         - --enable-leader-election
         image: controller:latest
+        imagePullPolicy: IfNotPresent
         name: manager
+        env:
+          - name: KUBERNETES_SERVICE_HOST
+            value: "172.17.0.2"
+          - name: KUBERNETES_SERVICE_PORT
+            value: "6443"
```

Comment:
I think these keys should go into an `apiserver-endpoint.patch.yaml`.
You can then include that patch in the `kustomization.yaml`